kubernetes/CoreDNS-k8s_version.md

```diff
@@ -5,7 +5,8 @@ This document records the CoreDNS version that was installed by kubeadm with eac
 
 | Kubernetes Version   |      CoreDNS version installed by kubeadm      |  Changes in CoreDNS from previous release to Kubernetes |
 |:--------------------:|:-------------------------:|:----------|
-| v1.20              |  [v1.7.0](https://github.com/coredns/coredns/releases/tag/v1.7.0) |  |
+| v1.21                |  [v1.8.0](https://github.com/coredns/coredns/releases/tag/v1.8.0) | https://github.com/coredns/coredns/blob/master/notes/coredns-1.7.1.md <br>https://github.com/coredns/coredns/blob/master/notes/coredns-1.8.0.md <br><br> ***---DEPRECATIONS---***  <br><br> <ul> <li>None</li> </ul> |
```

Comment:
One of the changes in 1.8.0 is moving to coredns/caddy
should that be mentioned here?

Reply:
No, it's transparent change, no user facing changes.